.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [ ubuntu, windows, macos ]
+
+    name: Nox on ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Python Setup
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
Nox should be run on all supported versions of python. You should use the matrix thing. Heres an example from one of my projects https://github.com/magpie-dev/hikari-crescent/blob/main/.github/workflows/pytest.yml